components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp

```diff
@@ -660,7 +659,6 @@ void MAMMicrophysics::run_impl(const double dt) {
   // Note: We are following the RRTMGP EAMxx interface to compute the zenith
   // angle. This operation is performed on the host because the routine
   // shr_orb_cosz_c2f has not been ported to C++.
-  auto ts2          = timestamp();
```

Comment:
@singhbalwinder I was puzzled by the `ts2` variable. Then I came to the conclusion that you are likely using ts2 (rather than the ts above) so that you do exactly what RRTMGP does, right? If that's the case, I will decide what to do with this mod (whether to keep it or roll it back) depending on what Ben says regarding the change of ts handling in rrtmgp.

Reply:
Yes, you are right. @odiazib added this code to mimic RRTMGP. We likely used `ts2` as `ts` was declared above and held a different value. It would be nice to unify these to avoid confusion.